socketserver.js

```diff
@@ -66,7 +67,7 @@ module.exports = (io, app) => {
       index = rooms.findIndex(x => x.roomID == socket.roomID);
       if(index >= 0){
         if(rooms[index].isFilled == true){
-          io.sockets.in(socket.roomID).emit('alone', { "message": "stranger is disconnected", "roomID": socket.roomID });
+          io.sockets.in(socket.roomID).emit('alone', { "title": "Stranger is disconnected!", "message": "Please click on 'New' button to connect to someone else.", "roomID": socket.roomID });
```

Comment:
rather than passing separate title and message, pass both of them as an object called ```message```. Nest the title and desciption inside.

Reply:
something like:
```
{ "message": { "title": "title", "description": "description" }, "roomID": socket.roomID }
```